client/client.js

```diff
@@ -0,0 +1,51 @@
+const minimist = require("minimist");
```

Comment:
Greate idea to differentiate between the client to handle the user API and the web crawler.